src/cfg/structured.rs

```diff
@@ -1,17 +1,17 @@
 use std::fmt::Display;
 
 use super::BasicBlock;
-use bril_rs::Argument;
+use bril_rs::{Argument, Code, EffectOps, Function, Instruction, Program};
 
-#[derive(Debug, PartialEq)]
+#[derive(Debug, PartialEq, Clone)]
 pub enum StructuredBlock {
     // Variable, then and else blocks
     Ite(String, Box<StructuredBlock>, Box<StructuredBlock>),
     Loop(Box<StructuredBlock>),
     Block(Box<StructuredBlock>),
     Sequence(Vec<StructuredBlock>),
     // how many layers of blocks / loops to break out of
-    Break(usize),
+    Break(i64),
```

Comment:
why did this need to change?

Reply:
Good catch, dumb reason I changed it back